lib/session/call-session.js

```diff
@@ -2601,6 +2604,26 @@ Duration=${duration} `
     }
   }
 
+  async _enableInbandDtmf(ep) {
```

Comment:
I think since we are going to call this on every call, but we rarely actually enable inband dtmf we should name it differently, maybe like "_enableInbandDtmfIfRequired"

Reply:
fixed